docs/whats-new/dotnet-docs-mod1.md

```diff
@@ -1,150 +1,129 @@
-- [--arch option doesn't imply self-contained](../core/compatibility/sdk/8.0/arch-option.md)
-
-## .NET Framework
-
-### Updated articles
-
-- [WCF Web HTTP Programming Model Overview](../framework/wcf/feature-details/wcf-web-http-programming-model-overview.md) - Fix broken links
+- [RuntimeHelpers.GetSubArray returns different type](../core/compatibility/core-libraries/9.0/getsubarray-return.md)
+- [Creating type of array of System.Void not allowed](../core/compatibility/core-libraries/9.0/type-instance.md)
+- [HttpListenerRequest.UserAgent is nullable](../core/compatibility/networking/9.0/useragent-nullable.md)
+- [ComponentDesigner.Initialize throws ArgumentNullException](../core/compatibility/windows-forms/9.0/componentdesigner-initialize.md)
+- [No exception if DataGridView is null](../core/compatibility/windows-forms/9.0/datagridviewheadercell-nre.md)
+- [DataGridViewRowAccessibleObject.Name starting row index](../core/compatibility/windows-forms/9.0/datagridviewrowaccessibleobject-name-row.md)
+- [Changes to nullability annotations (Windows Forms)](../core/compatibility/windows-forms/9.0/nullability-changes.md)
+- [BindingSource.SortDescriptions doesn't return null](../core/compatibility/windows-forms/9.0/sortdescriptions-return-value.md)
+- [Breaking changes in .NET 9](../core/compatibility/9.0.md)
+- [`dotnet workload` commands output change](../core/compatibility/sdk/9.0/dotnet-workload-output.md)
+- [Terminal logger is default](../core/compatibility/sdk/9.0/terminal-logger.md)
 
 ## .NET fundamentals
 
 ### New articles
 
-- [Reflection versus source generation in System.Text.Json](../standard/serialization/system-text-json/reflection-vs-source-generation.md)
-- [Compile-time options validation source generation](../core/extensions/options-validation-generator.md)
-- [How to read JSON as .NET objects (deserialize)](../standard/serialization/system-text-json/deserialization.md)
-- [Include fields](../standard/serialization/system-text-json/fields.md)
-- [Populate initialized properties](../standard/serialization/system-text-json/populate-properties.md)
-- [Build resilient HTTP apps: Key development patterns](../core/resilience/http-resilience.md)
-- [Introduction to resilient app development](../core/resilience/index.md)
-
-### Updated articles
-
-- [How to use source generation in System.Text.Json](../standard/serialization/system-text-json/source-generation.md) - Remaining STJ work for .NET 8
-- [Logging in C# and .NET](../core/extensions/logging.md) - Update logging docs
-- [Dispose Pattern](../standard/design-guidelines/dispose-pattern.md) - Add note that material may be out of date
-- [Tutorial: Create an item template](../core/tutorials/cli-templates-create-item-template.md) - Update template articles
-- [Tutorial: Create a template package](../core/tutorials/cli-templates-create-template-package.md) - Update template articles
-
-## .NET security
+- [MSTest design rules](../core/testing/mstest-analyzers/design-rules.md)
```

Comment:
An idea for organizing these files: instead of listing them alphabetically, group them by subfolder, and perhaps even add an H4 with the subfolder name.

Reply:
Logged here: https://github.com/dotnet/docs-tools/issues/276#issuecomment-1922152971